### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by IgorSusmelj**
### Changes

- Lots of smaller improvements on the docs
  - updated models in lightly docs
  - updated swav paper reference
  - added more detailed instructions for active learning using the docker
  - fixed some typos

### CODE REVIEW

File: docs/source/docker/advanced/active_learning.rst [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
 
+Learn more about the concept of active learning 
+:ref:`lightly-active-learning-scorers`.
+
+
+Create Embeddings
+--------------------------
 
+You can create embeddings using your own model. Just make sure the resulting
+`embeddings.csv` file matches the format of the one from
```
Comment (philippmwirth) [inline docs/source/docker/advanced/active_learning.rst:21]: Just do "matches the required format: "

File: docs/source/docker/advanced/active_learning.rst [hunk 2]
```diff
@@ -176,12 +188,18 @@ Your embeddings_al.csv should look similar to this:
...
-Step 3. Use the Lightly docker to perform a sampling on the scores.
----------------
-Run the docker and use the generated embedding file from the last step.
-Then perform an active learning sampling using the `CORAL` sampler.
-E.g. use the following bash script.
+Run Active Learning using the Docker
+---------------------------------------
+
+At this point you should have an `embeddings.csv` file with the active learning 
+scores in a column.
```
Comment (philippmwirth) [inline docs/source/docker/advanced/active_learning.rst:195]: maybe add: "in a column named `al_scores`"?

File: docs/source/docker/advanced/active_learning.rst [hunk 3]
```diff
@@ -176,12 +188,18 @@ Your embeddings_al.csv should look similar to this:
...
-E.g. use the following bash script.
+Run Active Learning using the Docker
+---------------------------------------
+
+At this point you should have an `embeddings.csv` file with the active learning 
+scores in a column. 
 
+We can now perform an active learning sampling using the `CORAL` sampler. In order
+to do the sampling on the `embeddings.csv` file we need to make this file
+accessible to the docker. An easy way is to use the `shared_dir` feature of the
```
Comment (philippmwirth) [inline docs/source/docker/advanced/active_learning.rst:199]: It's the only way at the moment so I don't konw if saying "An easy way ..." makes sense. What about "We can do this by using the `shared_dir` feature of the docker..."

File: docs/source/docker/overview.rst [hunk 4]
```diff
@@ -61,7 +63,7 @@ We worked hard to make this happen and are very proud to present you with the fo
   
   * 8 CPU cores or more
 
-  * 16GB free RAM
+  * 16GB free RAM (>32GB for datasets above 10m samples)
```
Comment (philippmwirth) [inline docs/source/docker/overview.rst:66]: It's even more, right? I required 40GB for 5 million samples.

I believe millions are indicated with a capital "M". Alternatively you can just write "million".

File: docs/source/getting_started/active_learning.rst [hunk 5]
```diff
@@ -56,6 +56,9 @@ Lightly makes use of the following concepts for active learning:
    different scores based on how certain the model is about the images. When
    performing a sampling, the scores are passed to the API so the sampler can use
    them with Coral.
+
+   Active learning scores are scalar values (per sample) between 0.0 and 1.0 whereas values
```
Comment (philippmwirth) [inline docs/source/getting_started/active_learning.rst:60]: whereas is incorrect here, use where instead

File: docs/source/getting_started/advanced.rst [hunk 6]
```diff
@@ -127,6 +127,13 @@ learning:
 
   - Check the documentation: :py:class:`lightly.models.barlowtwins.BarlowTwins`
 
+- `NNCLR: With a Little Help from My Friends: Nearest-Neighbor Contrastive Learning of Visual Representations, D. Dwibedi, 2021 <https://arxiv.org/abs/2104.14548>`_
+
+- `BYOL: Bootstrap your own latent: A new approach to self-supervised Learning, J. Grill, 2020 <https://arxiv.org/abs/2006.07733>`_
+
+- `SwAV: Unsupervised Learning of Visual Features by Contrasting Cluster Assignments, M. Caron, 2020 <https://arxiv.org/abs/2006.09882>`_
```
Comment (philippmwirth) [inline docs/source/getting_started/advanced.rst:134]: 😍

File: docs/source/docker/overview.rst (hunk 4 above)
Comment (IgorSusmelj) [reply docs/source/docker/overview.rst:66]: Changed to 64GB

**Review by philippmwirth — APPROVED** [review]

**Pull request merged**